manim/cli/new/group.py

```diff
@@ -0,0 +1,207 @@
+            cli_config["pixel_height"] = str(value[0])
+            cli_config["pixel_width"] = str(value[1])
+        else:
+            cli_config[key] = str(value)
+
+    with open(project_cfg_path, "w") as conf:
+        config.write(conf)
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    no_args_is_help=False,
+    epilog=EPILOG,
+    help="Create new projects.",
+)
+@click.argument("project_name", type=Path, required=False)
+@click.argument("template_name", required=False)
+@click.option(
+    "-d",
+    "--default",
+    "default_settings",
+    is_flag=True,
+    help="Default settings for project creation.",
+    nargs=1,
+)
+def project(default_settings, **args):
+    """Project subcommand creates a project inside of a 'new folder'. The name of the 'new folder' is passed as the first argument to the command.
+
+    Parameters
+    ----------
+        default_settings : :class:`bool`
+            Is True if -d or --default flag is passed.
+        **args : :class:`dict`
+            Dictionary of arguments passed to the 'project' command.
+    """
+    if args["project_name"]:
+        project_name = args["project_name"]
+    else:
+        project_name = click.prompt("Project Name", type=Path)
+
+    if args["template_name"]:
+        template_name = args["template_name"]
+    else:
+        # in the future when implementing a full template system. Choices are going to be saved in some sort of config file for templates
+        template_name = click.prompt(
+            "Template",
+            type=click.Choice(get_template_names(), False),
+            default="Default",
+        )
+
+    if project_name.is_dir():
+        console.print(
+            f"\nFolder [red]{project_name}[/red] exists. Please type another name\n"
+        )
+    else:
+        project_name.mkdir()
+        new_cfg = dict()
+        new_cfg_path = Path.resolve(project_name / "manim.cfg")
+
+        if not default_settings:
+            for key, value in CFG_DEFAULTS.items():
+                if key == "scene_names":
+                    if args["template_name"]:
+                        new_cfg[key] = args["template_name"]
+                    else:
+                        new_cfg[key] = value
+                elif key == "resolution":
+                    new_cfg[key] = select_resolution()
+                else:
+                    new_cfg[key] = click.prompt(f"\n{key}", default=value)
+
+            console.print("\n", new_cfg)
+            if click.confirm("Do you want to continue?", default=True, abort=True):
+                copy_template_files(project_name, template_name)
+                update_cfg(new_cfg, new_cfg_path)
+        else:
+            copy_template_files(project_name, template_name)
+            update_cfg(CFG_DEFAULTS, new_cfg_path)
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    no_args_is_help=True,
+    epilog=EPILOG,
+    help="Insert a scene to an existing file or a new file",
```

Comment:
I would use this help message in the docstring down below instead of here. See render command and how it's done there.
```suggestion
    help="Insert a scene to an existing file or a new file.",
```